apollo-normalized-cache-sqlite/src/commonMain/sqldelight/blob/com/apollographql/apollo3/cache/internal/blob/blob.sq

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE blobs (
+  key TEXT NOT NULL PRIMARY KEY,
+  blob BLOB NOT NULL
+) WITHOUT ROWID;
```

Comment:
The [WITHOUT ROWID](https://www.sqlite.org/withoutrowid.html) looks very promising. I'm hoping it can get us smaller/faster queries.